design.md

```diff
@@ -2,12 +2,50 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+We're still aiming this new language at CS5 students.
+We expect that they understand the picobot model and are thinking of it in
+terms of a state machine.
+Our syntax is also unique - we're not trying to mimic any other language
+(including natural language)
+
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
 
+Our goal with our design was to streamline the language. When writing code the
+user should feel like they're specifying the rules for a state machine in the
+most expressive way possible.
```

Comment:
I don't know enough about how JFLAP works, but doesn't it output some sort of compiled file?  So then you'd either have to implement syntax that matched that or, more likely imo, an external DSL that needs parsing?